Content.Server/Atmos/Components/BarotraumaComponent.cs

```diff
@@ -20,8 +20,11 @@ public class BarotraumaComponent : Component
     {
         public override string Name => "Barotrauma";
 
-        [DataField("damageType", required: true)]
-        private readonly string _damageType = default!;
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        [Robust.Shared.IoC.Dependency] private readonly IPrototypeManager _prototypeManager = default!;
```

Comment:
just import Robust.Shared.IoC and use [Dependency]

Reply:
Apparently System.Runtime.CompilerServices also has a [Dependency] attribute, so that throws an error, which is why I did it this way.